src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -8,6 +8,7 @@
 using System.Threading;
```

Comment:
It's really great that you got this new behavior into `RemoteRenderer` and not in the `Renderer` base class 👍